apps/dashboard/app/javascript/stylesheets/pinned_apps.scss

```diff
@@ -53,6 +53,42 @@
     border: 4px solid $launcher_border;
     border-radius: 10%;
   }
+/**
+  Pinned Apps border color overrides, to be used with the css_class tile property:
+  primary, secondary, success, info, warning, danger, light, and dark
+ */
+
+  a.primary {
+    border-color: var(--primary);
+  }
+
+  a.secondary {
+    border-color: var(--secondary);
+  }
+
+  a.success {
+    border-color: var(--success);
+  }
+
+  a.info {
+    border-color: var(--info);
+  }
+
+  a.warning {
+    border-color: var(--warning);
+  }
+
+  a.danger {
+    border-color: var(--danger);
+  }
```

Comment:
These are fine colors, but I feel like someone's going to want `red` or `green`. Can we add those too?

Reply:
Yes, that makes sense. Will add the CSS basic colors too:
https://www.w3.org/wiki/CSS/Properties/color/keywords